Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+    tissue_number text,
+    "sample-tissue" text,
+    cell_type_number text,
+    "sample-cell_type" text,
+    disease_number text,
+    "sample-disease" text,
+    "sample-description" text,
+    custom_number text,
+    "sample-custom" text,
+    ms_run_number text NOT NULL,
+    "ms_run-location" text NOT NULL,
+    "ms_run-instrument_ref" integer,
+    "ms_run-format" text,
+    "ms_run-id_format" text,
+    fragmentation_method_number text,
+    "ms_run-fragmentation_method" text,
+    scan_polarity_number text NOT NULL,
+    "ms_run-scan_polarity" text NOT NULL,
+    "ms_run-hash" text,
+    "ms_run-hash_method" text,
+    assay_number text NOT NULL,
+    assay text NOT NULL,
+    "assay-custom" text,
+    "assay-external_uri" text,
+    "assay-sample_ref" text,
+    "assay-ms_run_ref" text NOT NULL,
+    study_variable_number text NOT NULL,
+    study_variable text NOT NULL,
+    "study_variable-assay_refs" text NOT NULL,
+    "study_variable-average_function" text,
+    "study_variable-variation_function" text,
+    "study_variable-description" text NOT NULL,
+    "study_variable-factors" text,
+    "study-custom" text,
+    cv_number text NOT NULL,
+    "cv-label" text NOT NULL,
+    "cv-full_name" text NOT NULL,
+    "cv-version" text NOT NULL,
+    "cv-uri" text NOT NULL,
+    database_number text NOT NULL,
+    database text NOT NULL,
+    "database-prefix" text NOT NULL,
+    "database-version" text NOT NULL,
+    "database-uri" text NOT NULL,
+    derivatization_agent_number text,
+    derivatization_agent text,
+    "small_molecule-quantification_unit" text,
+    "small_molecule-identification_reliability" text,
+    id_confidence_measure_number text NOT NULL,
+    id_confidence_measure text NOT NULL,
+    "colunit-small_molecule" text,
+    "colunit-small_molecule_feature" text,
+    "colunit-small_molecule_evidence" text,
+    "ID" integer NOT NULL,
+    "mzTab-ID" text NOT NULL
+);
+
+
+ALTER TABLE "MzTab-M"."Metadata_metabolomics" OWNER TO postgres;
+
+--
+-- TOC entry 221 (class 1259 OID 17707)
+-- Name: Metadata_metabolomics_Internal_mzTab-ID_seq; Type: SEQUENCE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE SEQUENCE "MzTab-M"."Metadata_metabolomics_Internal_mzTab-ID_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab-M"."Metadata_metabolomics_Internal_mzTab-ID_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2969 (class 0 OID 0)
+-- Dependencies: 221
+-- Name: Metadata_metabolomics_Internal_mzTab-ID_seq; Type: SEQUENCE OWNED BY; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab-M"."Metadata_metabolomics_Internal_mzTab-ID_seq" OWNED BY "MzTab-M"."Metadata_metabolomics"."ID";
+
+
+--
+-- TOC entry 228 (class 1259 OID 18226)
+-- Name: Small Molecule Evidence (SME) Section; Type: TABLE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE TABLE "MzTab-M"."Small Molecule Evidence (SME) Section" (
+    "SME_ID" integer NOT NULL,
```

Comment:
The internal ID appears to be missing.  Also, the ID here does not match the text IDs you have used previously.  Is this supposed to be the internal ID?